net/interfaces/interfaces.go

```diff
@@ -278,13 +279,16 @@ type State struct {
 	// instead of Wifi. This field is not populated by GetState.
 	IsExpensive bool
 
-	// DefaultRouteInterface is the interface name for the machine's default route.
+	// DefaultRouteInterface is the interface name for the
```

Comment:
Is this used to bind values anywhere? I’m curious if we’re going from not binding to an interface to binding, which will change our behavior under all sorts of untested edge cases.

Reply:
Not on Windows yet, no. See `netns_windows.go` which does it by hand still. A future change can make it use this instead.

Reply:
so if I read this correctly, this PR is a no-op for edge cases. in that case, lgtm